app/config.js

```diff
@@ -13,6 +13,9 @@ module.exports = {
   // Enable or disable password protection on production
   useAuth: 'true',
 
+  // Enable or disable HTTPs / SSL on production
```

Comment:
dont think this is accurate - the code doesnt enable or disable HTTPS, it toggles a redirect